package.json

```diff
@@ -8,9 +8,9 @@
 		"build": "webpack --node-env production --progress",
 		"dev": "webpack --node-env development --progress",
 		"watch": "webpack --node-env development --progress --watch",
-		"test:unit": "TZ=UTC jest",
+		"test": "TZ=UTC jest",
```

Comment:
Can we keep the name as `test:unit` to be in line with the Nextcloud eco system?

Reply:
as far as I can see this is not something that's standardised and it's just redundant text since we don't have other commands. Just about as annoying as prefixing every css variable with "default".

E.G. both **photos** and **text** just use `npm run test`
**mail** uses `test:unit` and `test:unit:watch`

Reply:
`test:unit` is most likely legacy from using https://cli.vuejs.org

I'd prefer `test` name as it is more common in JS ecosystem and it has npm short run:

```bash
npm run test
# alias
npm test
```

We may old version as an alias aka:

```
"test": "TZ=UTC jest",
"test:unit": "npm test",
```